### PR TITLE
Alpha

### PR DESCRIPTION
**Pull request by drinkius**

### CODE REVIEW

**Review by GooGrand — APPROVED** [review]

File: Parser/WEB3/buyGTON.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { ethers } from 'ethers';
+import { FTMGTONSwapPath as path, spiritswaprouteraddress } from '../../config/config';
+import SpiritSwapRouterABI from './ABI/SpiritSwapRouter.json';
+import { validate } from './validate';
+
+// WFTM -> GTON
+
+declare const window: any;
+
+const buy = async (amount, gtonftmprice): Promise<string> =>
```
Comment (sham789) [inline Parser/WEB3/buyGTON.ts:10]: argument typing missed

File: config/config-test.ts [hunk 2]
```diff
@@ -1,33 +1,57 @@
...
 const network = '4002';
 const isDev = process.env.NODE_ENV === 'development';
 const isLive = 'true';
 const isTestnet = true;
 
+const WFTMAddress = "0xd0011de099e514c2094a510dd0109f91bf8791fa";
+const GTONAddress = "0xc4d0a76BA5909c8e764B67aCF7360f843FbACB2d";
+
+const spiritswaprouteraddress = '0x3c4925b50e337aecc2cf4b9e4767b43dcfbad286';
+const spiritswappooladdress = '0x131ee332febf8d2aa5f18f0d2c444e99411462e5';
```
Comment (sham789) [inline config/config-test.ts:10]: name constants in capitalized snake case.

**Review by sham789 — COMMENTED** [review]
please maintain rules of naming

File: config/config-test.ts (hunk 2 above)
Comment (drinkius) [reply config/config-test.ts:10]: Will fix it in the upcoming refactoring PR, @GooGrand please take a note

**Pull request merged**